src/store/asana/worker.ts

```diff
@@ -38,23 +42,27 @@ async function processAction(actions: Action[], errors: WorkerError[]): Promise<
       });
       break;
     }
+
     action.isProcessing = true;
+
     try {
       action.retries++;
       await action.func();
       actions.shift();
-      console.info("completed action");
     } catch (error: any) {
       actions.shift();
       if (
-        error.value.errors[0].message.indexOf("does not exist in parent") ===
-        -1
+        error.value
+        && error.value.errors
+        && error.value.errors.length >= 1
+        && error.value.errors[0].message.indexOf("does not exist in parent") === -1
```

Comment:
is there not a nullish synta error?.value

Reply:
Good call. Updated